### DISCUSSION TITLE
我们显卡cuda是11.4，lmdeploy能支持么？

### DISCUSSION THREAD

**Discussion by colinlzh** [InternLM/lmdeploy#1921] [Q&A]
显卡升级不太好升级，大批量的V100和A10显卡cuda都是11.4的。
我看文档说12.X支持，我们在12.X显卡上跑通了并且效果不错。
然后文档特别提到了11.8的安装方法，那11.4也按照11.8的安装方法来，然后改一改cuda版本变量么？

**Comment by lvhan028** [comment]
11.8包可以安装到 cu114 的环境中
